.github/workflows/main-mac-smoke-test.yaml

```diff
@@ -42,6 +42,7 @@ jobs:
           echo "${COSIGN_KEY}" > "$GITHUB_WORKSPACE/cosign.key"
         env:
           COSIGN_KEY: ${{ secrets.COSIGN_KEY }}
+      - run: make setup-ci-env
```

Comment:
Should `validate-ci` just have a prerequisite of `setup-ci-env`?

Reply:
Discussed offline.

Reply:
If done with current implementation, if a dev runs it locally, it would force reinstall of golangci-lint, or install it via another way if its already installed via brew or some other way. Adding a posix check to see if the command exists and installing golangci-lint if it is not already on the path.